src/lib/AnnotationControlsFSM.ts

```diff
@@ -39,6 +39,12 @@ export const stateModeMap = {
 export default class AnnotationControlsFSM {
     private currentState = AnnotationState.NONE;
 
+    public getState = (): AnnotationState => this.currentState;
+
+    public setState = (state: AnnotationState): void => {
```

Comment:
Maybe instead of exposing a setState, could we have an `initialState` in the constructor?